contracts/utils/cryptography/draft-ERC7739Utils.sol

```diff
@@ -0,0 +1,278 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+import {IERC5267} from "../../interfaces/IERC5267.sol";
+import {MessageHashUtils} from "./MessageHashUtils.sol";
+
+/**
+ * @dev Utilities to process https://eips.ethereum.org/EIPS/eip-7739[ERC-7739] typed data signatures
```

Comment:
The EIP is still not in that URL but I think it should be as soon as @Vectorized merges https://github.com/ethereum/ERCs/pull/441